creatingaDjangoApp.md

```diff
@@ -0,0 +1,51 @@
+# How to create a Django App
+
+A project is a collection of apps. An app is a web application which performs an small task like career options of a website or polling section.
+
+After creating our project we will `cd` into `mysite` or wherever `manage.py` file is present. Then type in following syntax:
+
+## Syntax for creating an app
+```
+Unix-python3 manage.py startapp APPNAME
+Windows-python manage.py startapp APPNAME
```

Comment:
Add a newline after `Unix-` and `Windows-`

Also four spaces in front of the command for the formatting